icx_omp_reduction/icx_reduction.c

```diff
@@ -0,0 +1,59 @@
+#define _POSIX_C_SOURCE 200809L
+#define START_TIMER(S) struct timeval start_ ## S , end_ ## S ; gettimeofday(&start_ ## S , NULL);
+#define STOP_TIMER(S,T) gettimeofday(&end_ ## S, NULL); T->S += (double)(end_ ## S .tv_sec-start_ ## S.tv_sec)+(double)(end_ ## S .tv_usec-start_ ## S .tv_usec)/1000000;
+
+#include "stdlib.h"
+#include "math.h"
+#include "sys/time.h"
+#include "xmmintrin.h"
+#include "pmmintrin.h"
+#include "omp.h"
+
+struct dataobj
```

Comment:
this is probably unnecessary

there are super quick things we can do to trim down this MFE, probably